ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -74,11 +77,32 @@
           condition-command: |
             #!/bin/bash
             set -x
-            # if the current branch is master then we don't run these tests
+            # if the target branch is master we DON'T RUN these tests.
```

Comment:
I don't think that's going to work since add_osds is not part of stable-3.0 and 3.1 so this is only valid for 3.2.